docs/UserGuide.md

```diff
@@ -3,12 +3,51 @@ layout: page
 title: User Guide
 ---
 
-RealEstatePro is a desktop app for managing contacts, optimized for real estate agents to manage their client's contacts and sales of properties.
+RealEstatePro is a desktop app for managing contacts, optimized for real estate agents to manage their client's contacts and sales of HDB properties.
+The app also includes various tools which aims to help agents in their work.
 
 # Reference
 
 Original AB3 User Guide: [link](https://se-education.org/addressbook-level3/UserGuide.html)
 
+#Table of contents
+- [Glossary](#glossary)
+- [Quick Start](#quick-start)
+- [Features](#features)
+- [General Commands](#general-commands)
+  - [Open Help Window: `help`](#open-help-window-help)
+  - [Listing all clients: `list`](#listing-all-clients--list)
+  - [Clearing all entries: `clear`](#clearing-all-entries--clear)
+  - [Exiting the program: `exit`](#exiting-the-program--exit)
+- [Client Management](#client-management)
+  - [Adding a client: `add`](#adding-a-client-add)
+  - [Editing a client: `edit`](#editing-a-client-edit)
+  - [Deleting a client: `delete`](#deleting-a-client-delete)
+  - [Favourite a client: `favourite`](#favourite-a-client-favourite)
+  - [Open Favourites window: `fw`](#open-favourites-window-fw)
+  - [Locating clients by name: `find`](#locating-clients-by-name-find)
+  - [Sorting clients: `sort`](#sorting-clients-sort)
+- [Tools](#tools)
+  - [Matching properties and preferences: `match`](#matching-properties-and-preferences-match)
+  - [Upload an image: `upload`](#upload-an-image--upload)
+  - [View image of client: `viewimage`](#view-image-of-client-viewimage)
+  - [Setting a reminder for a client: `remind`](#setting-a-reminder-for-a-client-remind)
+  - [Open Reminder window: `rm`](#open-reminder-window)
+  - [Displaying statistics: `stats`](#displaying-statistics-stats)
+- [Future Features](#future-features-coming-soon)
+- [Storage](#storage)
+  - [Saving the data](#saving-the-data)
+  - [Editing the data file](#editing-the-data-file)
+- [FAQ](#faq)
+- [Command Summary](#command-summary)
+- [Keyboard Shortcuts](#keyboard-shortcuts)
+
+## Glossary
+- **Users**: Real estate agents.
+- **Clients**: Customers of the real estate agents looking to buy or sell property.
+- **Buyers**: Clients that have engaged the real estate agent to help them buy a property.
+- **Sellers**: Clients that have engaged the real estate agent to help them sell their property.
```

Comment:
Glossary should be at the end of the document. Don't have to change this since I have fixed this in #187.

Reply:
Thanks for the help